schemas/log.yml

```diff
@@ -21,15 +21,18 @@
       description: >
         Offset of the beginning of the log event.
       example: 12
-    - name: message
+    - name: original
       type: keyword
       phase: 1
       example: "Sep 19 08:26:10 localhost My log"
       index: false
+      # TODO: Best would be to disable ignore_above completely
+      ignore_above: 32766
```

Comment:
I'm hesitant about the value of `ignore_above`, however. `ignore_above` does not truncate the value itself, it's a safety valve to avoid indexing values that are unexpectedly long in this field.

In other words, I don't think we would be likely to do exact match search or aggregations on an original log message that's this long. Log messages that are really short would be much more likely to be used this way, and as such I think 1024 would be enough.

Reply:
Agree with that. Removed the `ignore_above`.